cmd/env_test.go

```diff
@@ -22,3 +23,38 @@ func TestEnvironment_LogLevel(t *testing.T) {
 		})
 	}
 }
+
+func TestEnvironment_Batches(t *testing.T) {
+	cases := []struct {
+		lines   []string
+		batches map[string][]string
+	}{
+		{
+			lines:   []string{"foo:bar,baz"},
```

Comment:
Why invent a syntax?

The idea was to look YAML-ish in the workflow file:
```
with:
  batches: |
    foo:bar,baz
```

(or at least: not be JSON in the YAML, despite the temptation of  golang SDK).

But what's more YAML-ish than YAML?
```
with:
   batches: |
    foo: bar
    foz:
      baz
```